src/ORM/Query.php

```diff
@@ -1229,7 +1229,7 @@ public function delete(?string $table = null)
         $repository = $this->getRepository();
         $this->from([$repository->getAlias() => $repository->getTable()]);
 
-        return parent::delete();
+        return parent::delete($table);
```

Comment:
This will overwrite the table + alias set on line 1230 which I don't think is ideal.

Reply:
Then we should add a comment why the parent is called without the argument, as this is not usual :)